gradle/baseline.gradle

```diff
@@ -17,18 +17,37 @@ List<String> blacklistedBaselineProjects = [
         'profile-client-protobufs',
         'timestamp-impl']
 
+List<String> blacklistedErrorProneProjects = [
+        'atlasdb-console',
+        'atlasdb-dbkvs-hikari',
+        'atlasdb-jdbc',
+        'atlasdb-jdbc-tests',
+        'atlasdb-rocksdb',
+        'atlasdb-rocksdb-tests',
+        'atlasdb-service-server',
+        'cassandra-partitioner',
+        'commons-db',
+        'commons-executors',
+        'leader-election-impl',
+        'lock-impl',
+        'profile-client-protobufs',
+        'timestamp-impl']
+
 if (!blacklistedBaselineProjects.contains(project.name)) {
     apply plugin: 'com.palantir.baseline-checkstyle'
+} else if (!blacklistedErrorProneProjects.contains(project.name)) {
```

Comment:
This is not the intended behavior. We should add error-prone to all the projects who aren't in the second list. Replaced the 'else if' statement for an 'if' statement.